ArduCopter/sensors.cpp

```diff
@@ -166,6 +166,7 @@ void Copter::read_battery(void)
     // update motors with voltage and current
     if (battery.get_type() != AP_BattMonitor::BattMonitor_TYPE_NONE) {
         motors->set_voltage(battery.voltage());
+        AP_Notify::flags.battery_voltage = battery.voltage(); //for on-board oled display
```

Comment:
it may be for oled... it may be for something else. As far as this called is concerned, it's passing the info to the "notify" object. Notify object could have a buzzer, a oled display, a monitor, a text-to-speech interface or anything else. All of these "anything else" is not interesting to this caller.

It's very frequent that we later introduce more "backends"  and these comments just add noise to the codebase as they get outdated.